#Patch to fix a bug when building
#Jinja2's documentation using Sphinx
#Thanks to plevine457@verizon.net for patch
#Path url: Debian bug #527486
# Added by neurogeek@gentoo.org 07/13/2009
--- jinja2-2.1.1.orig/docs/conf.py
+++ jinja2-2.1.1/docs/conf.py
@@ -96,7 +96,7 @@
 #html_use_smartypants = True
 
 # use jinja2 for templates
-template_bridge = 'jinjaext.Jinja2Bridge'
+template_bridge = 'sphinx.jinja2glue.BuiltinTemplateLoader'
 
 # no modindex
 html_use_modindex = False
